### PR TITLE
Update the intellisense version

### LINKED ISSUE #6928: `GetDocFiles` may run more than once
`GetDocFiles` may run more than once:
https://github.com/dotnet/winforms/blob/725161305d1c0fe9a88914ae81466dc79dad3f38/eng/packageContent.targets#L23

This may cause build failures due to a race condition, e.g., [this build](https://dev.azure.com/dnceng/public/_build/results?buildId=1679927&view=results):
```
dotnet-install: Installation finished
  Determining projects to restore...
  Determining projects to restore...
  Determining projects to restore...
  Restored D:\a\_work\1\s\src\System.Windows.Forms\tests\UnitTests\WindowsFormsTests\WindowsFormsTests.proj (in 7.91 sec).
  Restored D:\a\_work\1\s\src\System.Windows.Forms\tests\InteropTests\NativeTests\NativeTests.proj (in 7.65 sec).
D:\a\_work\1\s\eng\packageContent.targets(55,5): error MSB3026: Could not copy "D:\a\_work\1\s\.packages\microsoft.private.intellisense\7.0.0-preview-20220316.2\IntellisenseFiles\windowsdesktop\1033\Accessibility.xml" to "D:\a\_work\1\s\artifacts\bin\docs\Accessibility.xml". Beginning retry 1 in 1000ms. The process cannot access the file 'D:\a\_work\1\s\artifacts\bin\docs\Accessibility.xml' because it is being used by another process.  [D:\a\_work\1\s\src\System.Windows.Forms\tests\InteropTests\NativeTests\NativeTests.proj]
##[error]eng\packageContent.targets(55,5): error MSB3026: (NETCORE_ENGINEERING_TELEMETRY=Restore) Could not copy "D:\a\_work\1\s\.packages\microsoft.private.intellisense\7.0.0-preview-20220316.2\IntellisenseFiles\windowsdesktop\1033\Accessibility.xml" to "D:\a\_work\1\s\artifacts\bin\docs\Accessibility.xml". Beginning retry 1 in 1000ms. The process cannot access the file 'D:\a\_work\1\s\artifacts\bin\docs\Accessibility.xml' because it is being used by another process. 
D:\a\_work\1\s\eng\packageContent.targets(55,5): error MSB3026: Could not copy "D:\a\_work\1\s\.packages\microsoft.private.intellisense\7.0.0-preview-20220316.2\IntellisenseFiles\windowsdesktop\1033\Accessibility.xml" to "D:\a\_work\1\s\artifacts\bin\docs\Accessibility.xml". Beginning retry 1 in 1000ms. The process cannot access the file 'D:\a\_work\1\s\artifacts\bin\docs\Accessibility.xml' because it is being used by another process.  [D:\a\_work\1\s\src\System.Windows.Forms\tests\InteropTests\NativeTests\NativeTests.proj]
##[error]eng\packageContent.targets(55,5): error MSB3026: (NETCORE_ENGINEERING_TELEMETRY=Restore) Could not copy "D:\a\_work\1\s\.packages\microsoft.private.intellisense\7.0.0-preview-20220316.2\IntellisenseFiles\windowsdesktop\1033\Accessibility.xml" to "D:\a\_work\1\s\artifacts\bin\docs\Accessibility.xml". Beginning retry 1 in 1000ms. The process cannot access the file 'D:\a\_work\1\s\artifacts\bin\docs\Accessibility.xml' because it is being used by another process. 
```

[BuildSrc-x86.zip](https://github.com/dotnet/winforms/files/8370456/BuildSrc-x86.zip)

/cc: @singhashish-wpf for visibility

### PR DESCRIPTION
**Pull request by RussKie**
* Update the intellisense version
* Avoid additional copy step
* Avoid race conditions

Closes #6928

## Test

Manually verified the content against a previously published microsoft.private.winforms NuGet package.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/6973)

### CODE REVIEW

**Comment by kant2002** [comment]
Nice!

File: eng/packageContent.targets [hunk 1]
```diff
@@ -3,62 +3,13 @@
   <PropertyGroup>
     <_DotnetApiDocsPackageName>Microsoft.Private.Intellisense</_DotnetApiDocsPackageName>
     <!-- Using ToLower below to unlock Linux build, even if it's unsupported. -->
-    <_DotnetApiDocsFilesRoot>$([MSBuild]::NormalizeDirectory('$(NuGetPackageRoot)', '$(_DotnetApiDocsPackageName.ToLower())', '$(MicrosoftPrivateIntellisenseVersion)', 'IntellisenseFiles', 'windowsdesktop'))</_DotnetApiDocsFilesRoot>
-    <_IntellisenseXmlDir>$([MSBuild]::NormalizeDirectory('$(ArtifactsBinDir)', 'docs'))</_IntellisenseXmlDir>
+    <_DotnetApiDocsFilesRoot>$([MSBuild]::NormalizeDirectory('$(NuGetPackageRoot)', '$(_DotnetApiDocsPackageName.ToLower())', '$(MicrosoftPrivateIntellisenseVersion)', 'IntellisenseFiles', 'windowsdesktop', '1033'))</_DotnetApiDocsFilesRoot>
```
[the remark this answers is not in the document]
Comment (RussKie) [reply eng/packageContent.targets:6]: This is what we have. And to be honest I never saw a localised intellisense.

[the remark this answers is not in the document]
Comment (dreddy-work) [reply eng/packageContent.targets:6]: Okay. i see nuget package itself is only English build.

**Review by dreddy-work — APPROVED** [review]

File: eng/packageContent.targets (hunk 1 above)
[the remark this answers is not in the document]
Comment (RussKie) [reply eng/packageContent.targets:6]: I've enquired about this, unfortunately the repo that produces the intellisense can't be registered with the BAR system, and thus flow via the maestro service.

**Pull request merged**